examples/dt/uni07eta/README.md

```diff
@@ -0,0 +1,258 @@
+
+This document contains a list of integration test suites that would be
+executed against the below specified topology of Red Hat OpenStack Services
+on OpenShift. It also contains a collection of custom resources (CRs) for
+deploying the test environment.
+
+## Purpose
+
+This topology is used for executing integration tests that evaluate the
+`default` backends of the below mentioned services.
+
+## Environment
+
+### Nodes
+
+| Role | Machine Type | Count |
+| ---- | ------------ | ----- |
+| Compact OpenShift | vm | 3 |
+| OpenStack Compute | vm | 2 |
+| Networker         | vm | 3 |
+| Test nodes        | vm | 2 |
+
+### Networks
+
+| Name | Type | Interface |
+| ---- | ---- | --------- |
+| Provisioning | untagged | nic1 |
+| Machine | untagged | nic2 |
+| RH OSP | trunk | nic3 |
+
+#### Networks in RH OSP
+
+| Name | Type |
+| ---- | ---- |
+| ctlplane | untagged |
+| internalapi | VLAN tagged |
+| Storage | VLAN tagged |
+| Tenant | VLAN tagged |
+
+### Services, enabled features and configurations
+
+| Service          | configuration            | Lock-in coverage?  |
+| ---------------- | ------------------------ | ------------------ |
+| Cinder           | iSCSI (netapp) multipath | Must have          |
+| Cinder Backup    | SwiftBackupDriver        | Must have          |
+| Glance           | Cinder (iSCSI)           | Must have          |
+| Swift            | (default)                | Must have          |
+| Octavia          | OVN provider             | Must have          |
+| Horizon          | N/A                      | Must have          |
+| Barbican         | (default)                | Must have          |
+| Telemetry        |                          | Must have          |
+| Ironic           |                          | Must have          |
+| Neutron          | vlan provider (dvr)      | Must have          |
+
+#### Support services
+
+The following table lists services which are not the main focus of the testing
+(which may be covered by additional scenarios), but are required for the DT to
+work properly and can be deployed with any/default configuration.
+
+| Service          | Reason            |
+| ---------------- |------------------ |
+| Nova             | needed by scenario testing |
+| Keystone         | needed by all services     |
+
+### Additional configuration
+
+- Always-on, default services and features: TLSe
+- Two additional fake baremetal nodes
+- Availability zones for OVN (zone-1 & zone-2)
+- Logical volume with the name cinder-volumes exists on a OpenShift node.
+- iSCSI service is enabled on all OpenShift nodes.
+- Multipath service is enabled on all OpenShift nodes.
+
+#### iSCSI
+
+It is assumed *iSCSI* services are enabled in all nodes participating in the
+Red Hat OpenShift cluster. If not, a `MachineConfig` similar to the below one
+is applied. The node would be *rebooted* after applying the configuration.
+
+```YAML
+---
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
```

Comment:
I have not tried `MachineConfig`. I'm relying on people in the cinder group to review this part.

Reply:
Hello, this is a copy-paste of what we were told to use as a baseline – please ask the questions there: https://github.com/openstack-k8s-operators/architecture/pull/83

Reply:
This is working, according @psathyan